.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "eslint.format.enable": true,
+  "[javascript]": { "editor.defaultFormatter": "dbaeumer.vscode-eslint" },
+  "[typescript]": { "editor.defaultFormatter": "dbaeumer.vscode-eslint" }
+}
```

Comment:
the default formatter should be prettier

Reply:
We are running Prettier as a Rule through ESLint

Reply:
yeah, but won't running eslint as a formatter also "fix" issues automatically? I'd like to avoid that implicit behaviour